application/templates/enterdob.html

```diff
@@ -0,0 +1,53 @@
+{% extends "base/lr_base_layout.html" %}
+
+{% block content %}
+  <div class="main" id="content">
+    {{ super() }}
+    <div>&nbsp;</div>
```

Comment:
This was added for formatting it will be refactored as part of a later tech debt spike